.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build Paper
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  unittest:
+    name: Paper build
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - 'ubuntu-18.04'
+        latex-entry:
+          - 'sample-bachelor'
+          - 'sample-kaitireport'
+          - 'sample-master'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 10
+      - name: Set up system dependences on linux
+        if: ${{ runner.os == 'Linux' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y tree cloc make
+      - name: Load the font requirements
+        run: |
+          git clone "https://github.com/HansBug/BUAAthesis_fonts" -b master local_fonts
```

Comment:
How about download the fonts from

```
wget https://dl.freefontsfamily.com/download/Times-New-Roman-Font/ -o times-new-roman.zip
wget https://www.wfonts.com/download/data/2014/06/01/stxingkai/stxingkai.zip
```

Reply:
We could add a link to the readme to tell users that they can download those two fonts from above links.

Reply:
Completed.